docs/platform/api/upgrade-to-linode-api-v4.md

```diff
@@ -0,0 +1,190 @@
+---
+author:
+  name: Jared Kobos
+  email: docs@linode.com
+description: 'The new Linode API includes a number of additional features and changes from previous API versions. This guide is intended to help existing users upgrade to the new API.'
+keywords: ["api","linode api"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2018-05-08
+modified_by:
+  name: Linode
+title: 'Upgrade to the New Linode API'
+published: 2018-05-08
+external_resources:
+ - '[Linode API Documentation](https://developers.linode.com)'
+---
+
+Version 4 of the Linode API, which is now in general release, is a significant departure from previous versions. If you have existing code that uses an older API version, you will have to adapt it in order to work with version 4. This guide will discuss the key changes in the Linode API and give examples to help you modify your code to take advantage of the new features in the current version.
+
+If you have not used previous versions and are looking to get started with the current version, please see the [API documentation](https://developers.linode.com) or our [Getting Started with the Linode API guide](/docs/platform/api/getting-started-with-the-linode-api/).
+
+## General Changes
+
+* The Linode API now uses a standard RESTful architecture. HTTP verbs have predictable results across the API:
+
+    | Verb  |  Result |
+    |---|---|
+    | GET  | used to retrieve information about a resource.  |
+    | POST | create a new instance of a resource.  |
+    | PUT  | update a resource.  |
+    | DELETE  | remove a resource from your account.  |
+
+* Previous versions of the API allowed users to submit request and method parameters via URL parameters; this is no longer supported. All information needed for a request must be included in the body of a POST or PUT request.
+
+* Error codes also follow standard RESTful format; the custom error codes from previous versions are no longer used or supported.
```

Comment:
Maybe a brief overview, like "2xx is success, 4xx is bad input, 5xx is server error" would make this more clear.